core/services/feeds/models.go

```diff
@@ -0,0 +1,87 @@
+package feeds
+
+import (
+	"database/sql/driver"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/lib/pq"
```

Comment:
We are trying to phase our `pq` in favour of pgx, let's try not to introduce any new references to it.

Reply:
I believe this is still needed here because I'm storing an array and I need the field to implement Scan/Value.

Do you have any suggestions on how to avoid `pq` here?

Reply:
Not really, I think we'll have to just live with it